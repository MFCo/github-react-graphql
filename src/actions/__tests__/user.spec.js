import configureStore from 'redux-mock-store';
import act from '../../actions';

const middlewares = [];
const mockStore = configureStore(middlewares);

it('should dispatch action to update user', () => {

  const initialState = {};
  const store = mockStore(initialState);

  store.dispatch(act.updateUser({
    newName: 'Jhon Doe',
    newUser: 'JDoe',
    newAvatar: 'image.jpg',
    newBio: 'hi!',
    newCompany: 'company', 
    newLocation: 'London, Spain',
    newMail:'gmail@jhondoe.com',
    newWebsite: 'com.ar.jhon'
  }));

  const actions = store.getActions();
  expect(actions).toMatchSnapshot();
});