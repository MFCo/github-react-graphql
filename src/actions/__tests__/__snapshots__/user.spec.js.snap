// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should dispatch action to update user 1`] = `
Array [
  Object {
    "payload": Object {
      "newAvatar": "image.jpg",
      "newBio": "hi!",
      "newCompany": "company",
      "newLocation": "London, Spain",
      "newMail": "gmail@jhondoe.com",
      "newName": "Jhon Doe",
      "newUser": "JDoe",
      "newWebsite": "com.ar.jhon",
    },
    "type": "UPDATE_USER",
  },
]
`;
